lib/identity_cors.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class IdentityCors
+  FEDERALIST_REGEX = %r{^https://federalist-[0-9a-f-]+\.app\.cloud\.gov$}
+  STATIC_SITE_ALLOWED_ORIGINS = [
+    'https://www.login.gov',
+    'https://login.gov',
+    'https://handbook.login.gov',
+    FEDERALIST_REGEX,
+  ].freeze
+
+  def self.allowed_origins_static_sites
+    return STATIC_SITE_ALLOWED_ORIGINS unless Rails.env.development? || Rails.env.test?
+    allowed_origins = STATIC_SITE_ALLOWED_ORIGINS.dup
+    allowed_origins << %r{https?://localhost(:\d+)?$}
+    allowed_origins << %r{https?://127\.0\.0\.1(:\d+)?$}
+
+    allowed_origins
+  end
+
+  def self.allowed_origins_api(source)
```

Comment:
since this is a boolean result, let's name it like a predicate? Also maybe rename to focus more on redirect URIs?

```suggestion
  def self.allowed_redirect_uri?(source)
```

Reply:
If we do, can we also change the first line to return a boolean value rather than `nil`?

https://github.com/18F/identity-idp/blob/39877c9d18ad393b9437ab792f18bb2f18c1dd9f/lib/identity_cors.rb#L22

Reply:
I'm actually not sure what happens for nil/false/true in this, I'll do some testing

Reply:
Changed to returning `false`, behavior seems to be identical between `nil` and `false`